Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -234,6 +234,11 @@
 #endif
 
 #if ENABLE(SCREEN_TIME)
+#if PLATFORM(MAC) || PLATFORM(IOS_FAMILY)
+@interface STWebpageController (Staging_138865295)
+@property (nonatomic, copy) STWebHistoryProfileIdentifier profileIdentifier;
+@end
+#endif
```

Comment:
Nit - you don't need the guards here because `PLATFORM(MAC) || PLATFORM(IOS_FAMILY)` is equivalent to `PLATFORM(COCOA)`, but we're already in an `mm` file.

```suggestion
@interface STWebpageController (Staging_138865295)
@property (nonatomic, copy) STWebHistoryProfileIdentifier profileIdentifier;
@end
```

This should also (ideally) be moved into a separate `ScreenTimeSPI.h` header.